content/posts/newsletter/2021-09-05.md

```diff
@@ -0,0 +1,197 @@
+
+Image: https://gajim.org/img/news/workspaces-preview-1.png
+
+[Libervia progress note 2021-W31](https://www.goffi.org/b/libervia-progress-note-2021-w31-Ua14) is out with information about Docker integration, the translation portal and the first 0.8.0 beta. It also has plenty of details about the work done on the ActivityPub Gateway project (grant announced in the [April '21 newsletter](https://xmpp.org/2021/04/newsletter-04-april/)) with SQL, DBus, PubSub and with new and updated XEPs.
+
+While the Snikket iOS client app was just released, read more below, the behind the scenes development continues. In the latest blog post, Matthew Wild announces that [the expert folk at Simply Secure will be performing a usability audit of the current app](https://snikket.org/blog/simply-secure-collaboration/), as well as conducting usability testing thanks to funding from the OTF’s Usability Lab. The analysis will help to improve the UX of the iOS app and Snikket as a whole.
+
+Communiqué is a new XMPP client from the [Mellium Co-op](https://opencollective.com/mellium) team. It was [announced](https://opencollective.com/mellium/updates/introducing-communique) this month and presented at the XMPP Office Hours (unfortunately recording did not work out). The source code can be found [in the repository](https://github.com/mellium/xmpp/).
+
+Image: https://opencollective-production.s3.us-west-1.amazonaws.com/85c589a0-f462-11eb-af6c-e7a0da2f7bbc.png
+
+Monal 5.0.1 is now available [for both iOS and macOS](https://monal.im/blog/monal-5-0-1-synchronized-builds-and-bugfixes/) bringing mostly corrections and more polish over the previously major release.
+
+JSXC Openfire plugin gets a [4.3.1-1 release](https://discourse.igniterealtime.org/t/jsxc-openfire-plugin-4-3-1-1-released/90583), with mostly bug fixes and improvements from the JSXC project.
+
+After so many months of waiting [the Snikket iOS app is now publicly released](https://snikket.org/blog/snikket-ios-public-release/). Snikket server admins can add the app to the invitations pages to have Apple users easily find it. If you are not running Snikket you can still use the app (you can use credentials directly) but do read the blog post to know what you need to add to your Prosody instance (invitations modules) or what limitations you might experience using any other server software.
+ 
+Image: App store
+
+## Servers
+
+Prosody 0.11.10 has been [released](https://blog.prosody.im/prosody-0.11.10-released/) with a fix for CVE-2021-37601 and some minor changes. Prosody developers recommend server admins to upgrade in order to fix the remote information disclosure issue.
+
+## Libraries
+
+The Mellium Dev Communiqué for August includes updates to the Mellium XMPP library as well as the new Communiqué instant messaging client (discussed more below). The biggest updates this month are MAM and ad-hoc commands support! You can read more here:  https://opencollective.com/mellium/updates/dev-communique-for-august-2021
```

Comment:
"discussed more above" maybe? I don't know if it got shuffled around or I mixed things up somehow before, but this doesn't seem right. Thanks for fixing it and adding it back!

Reply:
4c003d0